README.md

```diff
@@ -135,6 +139,39 @@ Wait until the operator discovered and executed the change, then look at your AW
 
 ![Update stack](docs/img/stack-update.png)
 
+## Tags
+
+You may want to assign tags to your CloudFormation stacks. The tags added to a CloudFormation stack will be propogated to the managed resources. This feature may be useful in multiple cases, for example to distinguish resources at billing report. Current operator provides two ways to assign tags:
+- `global-tags` command line argument or `GLOBAL_TAGS` environment variable which allows to set global tags for all resources managed by the operator. This option accepts JSON format where every key is a tag name and value is a tag value. For example '{"foo": "fooValue", "bar": "barValue"}'
+- `tags` parameter at kubernetes resource spec:
+```yaml
+apiVersion: cloudformation.linki.space/v1alpha1
+kind: Stack
+metadata:
+  name: my-bucket
+spec:
+  tags:
+    foo: dataFromStack
+  template: |
+    ---
+    AWSTemplateFormatVersion: '2010-09-09'
+
+    Resources:
+      S3Bucket:
+        Type: AWS::S3::Bucket
+        Properties:
+          VersioningConfiguration:
+            Status: Enabled
+```
+
+Resource-specific tags have precedence over global tags. Thus if a tag is defined at command-line arguments and for a `Stack` resource, the value from the `Stack` resource will be used. 
+
+If we run the operation and a `Stack` resource with the describved above examples, we'll see such picture:
```

Comment:
Typo: `describved`

Reply:
Hey @davidxjohnson , thank you, fixed. Along with some other texting issues.